SpriteKit/MendeluSpriteKit/Game/WelcomeScreen.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  WelcomeScreen.swift
+//  MendeluSpriteKit
+//
+//  Created by Raul Batista on 08.11.2022.
+//
+
+import Foundation
+import SpriteKit
+
+final class WelcomeScreen: SKScene {
+    // MARK: Properties
+    private var newGameButton: TouchNode?
+    
+    var newGameButtonTapped: (() -> Void)?
+}
+
+// MARK: Public API
+extension WelcomeScreen {
+    override func didMove(to view: SKView) {
+        newGameButton = children.first(where: { $0 is TouchNode }) as? TouchNode
+        newGameButton?.tapAction = { [weak self] in
+            self?.newGameButtonTapped?()
+        }
+    }
+    
+    func setBackgroundImage(_ image: UIImage) {
```

Comment:
I think it makes sense to have this function private and call it in `sceneDidLoad()`.

Reply:
for me definitely private method 👍